docs/querying/query-context.md

```diff
@@ -92,7 +92,7 @@ include "selector", "bound", "in", "like", "regex", "search", "and", "or", and "
 - All aggregators must offer vectorized implementations. These include "count", "doubleSum", "floatSum", "longSum", "longMin",
  "longMax", "doubleMin", "doubleMax", "floatMin", "floatMax", "longAny", "doubleAny", "floatAny", "stringAny",
  "hyperUnique", "filtered", "approxHistogram", "approxHistogramFold", and "fixedBucketsHistogram" (with numerical input). 
-- No virtual columns.
+- All virtual columns must offer vectorized implementations.
```

Comment:
There's only one built-in virtual column: "expression". So, we should mention when that specific type of virtual column can be vectorized. (Does it depend on the type of expression? Types of input? Functions used? etc)

Reply:
Ah, I was planning on filling out a list of supported expressions but figured the list was sort of volatile since I have a few PRs and branches in flight and was originally going to do this later. I was planning to wait for #10429 to go in before I fixup this PR (to make sure changing the default to false doesn't cause any ill effects to tests) so I can go ahead and fill out this list to include the stuff from #10401 and #10429 in this PR at least.

Reply:
added docs explaining what is currently supported